README.md

```diff
@@ -1,62 +1,166 @@
-![](https://foundation-prod-assetspublic53c57cce-8cpvgjldwysl.s3-us-west-2.amazonaws.com/assets/logo-256.png)
 
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 [![All Contributors](https://img.shields.io/badge/all_contributors-109-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
-# Welcome to Dendron!
+# Dendron
 
-## What is Dendron?
+![Dendron Logo](https://foundation-prod-assetspublic53c57cce-8cpvgjldwysl.s3-us-west-2.amazonaws.com/assets/logo-256.png)
 
-A personal knowledge management tool that **grows** as you do.
+Dendron is an **open-source, local-first, markdown-based, note-taking tool**. It's a personal knowledge management solution (PKM) built specifically for developers and integrates natively with IDEs like [VSCode](https://code.visualstudio.com/).
 
-Dendron is a local-first, markdown based, hierarchical note-taking application built on top of [VSCode](https://code.visualstudio.com/) and [friends](./ACKNOWLEDGEMENTS.md).
+## Why
 
-Whether you organize your notes with [PARA](https://fortelabs.co/blog/para/) or [Zettelkasten](https://zettelkasten.de/), link notes together like [roam](https://roamresearch.com/), or simply create notes in an ad-hoc manner, Dendron can help.
+Most PKM tools will help you create notes but hit a wall trying to retrieve them past some threshold. That threshold varies with the tool — 100, 1000, 10 000, virtually everything stops working past 10k notes unless the human was super diligent about organizing their knowledge. Past this threshold, entropy wins and every query becomes a keyword search and scrolling through pages of results.
 
-Think of it as a [second brain](https://www.buildingasecondbrain.com/) to help you **make sense** of all the information you care about.
+> Dendron's mission is to help **humans organize, find, and work with any amount of knowledge**.
 
-Some potential use cases include:
+It not only helps you create notes, but also retrieve them - retrieval works as well with ten notes as it does with ten thousand.
 
-- project management
-- personal knowledge base of things you've learned
-- CRM to keep track of clients, friends and enemies
-- recipe book
+## How
 
-Click [here](https://youtu.be/6rwWUalr9Ac) to see an example of Dendron in action.
+Dendron builds on top of the past five decades of language design and developer tooling. We apply the key lessons from software to the management of general knowledge. We make managing general knowledge more like managing code, and your PKM more like an IDE for general knowledge.
```

Comment:
`of *programming* language design`

"We make managing general knowledge like managing code and your PKM like an IDE."